### PR TITLE
Misc ng gear

### PR DESCRIPTION
**Pull request by phawk**
- Ability to create a new story
- shortDate filter for overall projects page displaying 'today', 'Yesterday', '30th Oct' etc.

### CODE REVIEW

File: app/assets/javascripts/templates/directives/markdown_input.jst.ejs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="text-box">
+  <textarea class="atwho" ng-model="content" placeholder="{{ filler }}"></textarea>
```
Comment (colinyoung87) [inline app/assets/javascripts/templates/directives/markdown_input.jst.ejs]: is the ng-model always going to be `content`? or does that need to be customisable?

Comment (phawk) [inline app/assets/javascripts/templates/directives/markdown_input.jst.ejs]: yeah, content is just `$scope.content` and thats whatever is passed into the directive

Comment (colinyoung87) [inline app/assets/javascripts/templates/directives/markdown_input.jst.ejs]: just seen that and was about to say haha

File: app/controllers/api/v1/stories_controller.rb [hunk 2]
```diff
@@ -33,13 +33,14 @@ def collaborators
       end
 
       api! "Create a new story"
-      param :project_id, String, desc: "Project ID or slug", required: true
+      param :"stpry[project_id]", String, desc: "Project ID or slug", required: true
```
Comment (colinyoung87) [inline app/controllers/api/v1/stories_controller.rb]: just a spelling mistake on story there

**Comment by colinyoung87** [comment]
:+1:

**Pull request merged**